File: client/iih_stub/iih.ci_stub/iih.ci.ord/ciordems/d/EmsItemInCons.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.ciordems.d
{
    /// <summary>
    /// EmsItemInCons 的摘要说明。
    /// </summary>
    public class EmsItemInCons : BaseDTO {

        public EmsItemInCons() {
 
        }

        /// <summary>
        /// 主键
        /// </summary>
		public string Id_emsitemincons {
            get { return getAttrVal<string>("Id_emsitemincons",null); }
            set { setAttrVal<string>("Id_emsitemincons", value); }
        }

        /// <summary>
        /// 排序号
        /// </summary>
		public int? Sortno {
            get { return getAttrVal<int?>("Sortno",null); }
            set { setAttrVal<int?>("Sortno", value); }
        }

        /// <summary>
        /// 受邀机构id
        /// </summary>
		public string Id_org {
            get { return getAttrVal<string>("Id_org",null); }
            set { setAttrVal<string>("Id_org", value); }
        }

        /// <summary>
        /// 受邀机构
        /// </summary>
		public string Name_org {
            get { return getAttrVal<string>("Name_org",null); }
            set { setAttrVal<string>("Name_org", value); }
        }

        /// <summary>
        /// 受邀科室id
        /// </summary>
		public string Id_dep_emp {
            get { return getAttrVal<string>("Id_dep_emp",null); }
            set { setAttrVal<string>("Id_dep_emp", value); }
        }

        /// <summary>
        /// 受邀科室
        /// </summary>
		public string Name_dep_emp {
            get { return getAttrVal<string>("Name_dep_emp",null); }
            set { setAttrVal<string>("Name_dep_emp", value); }
        }

        /// <summary>
        /// 受邀职称编码
        /// </summary>
		public string Sd_emp_title {
            get { return getAttrVal<string>("Sd_emp_title",null); }
            set { setAttrVal<string>("Sd_emp_title", value); }
        }

        /// <summary>
        /// 受邀职称id
        /// </summary>
		public string Id_emp_title {
            get { return getAttrVal<string>("Id_emp_title",null); }
            set { setAttrVal<string>("Id_emp_title", value); }
        }

        /// <summary>
        /// 受邀职称
        /// </summary>
		public string Name_emp_title {
            get { return getAttrVal<string>("Name_emp_title",null); }
            set { setAttrVal<string>("Name_emp_title", value); }
        }

        /// <summary>
        /// 受邀人id
        /// </summary>
		public string Id_emp_doctor {
            get { return getAttrVal<string>("Id_emp_doctor",null); }
            set { setAttrVal<string>("Id_emp_doctor", value); }
        }

        /// <summary>
        /// 受邀人
        /// </summary>
		public string Name_emp_doctor {
            get { return getAttrVal<string>("Name_emp_doctor",null); }
            set { setAttrVal<string>("Name_emp_doctor", value); }
        }

        /// <summary>
        /// 应答时间
        /// </summary>
		public DateTime? Dt_response {
            get { return getAttrVal<FDateTime>("Dt_response",null); }
            set { setAttrVal<FDateTime>("Dt_response", value); }
        }

        /// <summary>
        /// 应答标志
        /// </summary>
		public bool? Fg_response {
            get { return getAttrVal<FBoolean>("Fg_response",null); }
            set { setAttrVal<FBoolean>("Fg_response", value); }
        }

        /// <summary>
        /// 应答人id
        /// </summary>
		public string Id_emp_response {
            get { return getAttrVal<string>("Id_emp_response",null); }
            set { setAttrVal<string>("Id_emp_response", value); }
        }

        /// <summary>
        /// 应答人
        /// </summary>
		public string Name_emp_response {
            get { return getAttrVal<string>("Name_emp_response",null); }
            set { setAttrVal<string>("Name_emp_response", value); }
        }

        /// <summary>
        /// 会诊项目id
        /// </summary>
		public string Id_srv {
            get { return getAttrVal<string>("Id_srv",null); }
            set { setAttrVal<string>("Id_srv", value); }
        }

        /// <summary>
        /// 会诊项目
        /// </summary>
		public string Name_srv {
            get { return getAttrVal<string>("Name_srv",null); }
            set { setAttrVal<string>("Name_srv", value); }
        }

        /// <summary>
        /// 会诊类型编码
        /// </summary>
		public string Sd_constp {
            get { return getAttrVal<string>("Sd_constp",null); }
            set { setAttrVal<string>("Sd_constp", value); }
        }

        /// <summary>
        /// 会诊类型id
        /// </summary>
		public string Id_constp {
            get { return getAttrVal<string>("Id_constp",null); }
            set { setAttrVal<string>("Id_constp", value); }
        }

        /// <summary>
        /// 会诊类型
        /// </summary>
		public string Name_constp {
            get { return getAttrVal<string>("Name_constp",null); }
            set { setAttrVal<string>("Name_constp", value); }
        }


        /// <summary>
        /// 版本号
        /// </summary>
        public DateTime? Sv
        {
            get { return getAttrVal<DateTime?>("Sv", null); }
            set { setAttrVal<DateTime?>("Sv", value); }
        }

        /// <summary>
        /// 医保
        /// </summary>
        public string Id_haeth
        {
            get { return getAttrVal<string>("Id_hp", null); }
            set { setAttrVal<string>("Id_hp", value); }
        }

        /// <summary>
        /// 医保名称
        /// </summary>
        public string Name_heath
        {
            get { return getAttrVal<string>("Name_hp", null); }
            set { setAttrVal<string>("Name_hp", value); }
        }
        /// <summary>
        /// 服务医保类型
        /// </summary>
        public string Id_hpsrvtp
        {
            get { return getAttrVal<string>("Id_hpsrvtp", null); }
            set { setAttrVal<string>("Id_hpsrvtp", value); }
        }

        /// <summary>
        /// 服务医保类型编码
        /// </summary>
        public string Sd_hpsrvtp
        {
            get { return getAttrVal<string>("Sd_hpsrvtp", null); }
            set { setAttrVal<string>("Sd_hpsrvtp", value); }
        }

        /// <summary>
        /// 医疗单来源
        /// 内部类型：[OrSrvAgentInfoDO]
        /// </summary>
        public int? Eu_sourcemd
        {
            get { return getAttrVal<int?>("Eu_sourcemd", null); }
            set { setAttrVal<int?>("Eu_sourcemd", value); }
        }

        /// <summary>
        /// 所属服务来源
        /// </summary>
        public string Id_srv_src
        {
            get { return getAttrVal<string>("Id_srv_src", null); }
            set { setAttrVal<string>("Id_srv_src", value); }
        }
        /// <summary>
        /// 计价状态
        /// </summary>
        public string Priby
        {
            get { return getAttrVal<string>("Priby", null); }
            set { setAttrVal<string>("Priby", value); }
        }


        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[] { "Id_emsitemincons", "Sortno", "Id_org", "Name_org", "Id_dep_emp", "Name_dep_emp", "Sd_emp_title", "Id_emp_title", "Name_emp_title", "Id_emp_doctor", "Name_emp_doctor", "Dt_response", "Fg_response", "Id_emp_response", "Name_emp_response", "Id_srv", "Name_srv", "Sd_constp", "Id_constp", "Name_constp", "Sv", "Id_hp", "Name_hp", "Id_hpsrvtp", "Sd_hpsrvtp" };
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.ciordems.d.EmsItemInCons";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/ciordems/d/EmsItemInCons.java
package iih.ci.ord.ciordems.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 会诊受邀科室项目 DTO数据 
 * 
 */
public class EmsItemInCons extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 主键
	 * @return String
	 */
	public String getId_emsitemincons() {
		return ((String) getAttrVal("Id_emsitemincons"));
	}
	/**
	 * 主键
	 * @param Id_emsitemincons
	 */
	public void setId_emsitemincons(String Id_emsitemincons) {
		setAttrVal("Id_emsitemincons", Id_emsitemincons);
	}
	/**
	 * 排序号
	 * @return Integer
	 */
	public Integer getSortno() {
		return ((Integer) getAttrVal("Sortno"));
	}
	/**
	 * 排序号
	 * @param Sortno
	 */
	public void setSortno(Integer Sortno) {
		setAttrVal("Sortno", Sortno);
	}
	/**
	 * 受邀机构id
	 * @return String
	 */
	public String getId_org() {
		return ((String) getAttrVal("Id_org"));
	}
	/**
	 * 受邀机构id
	 * @param Id_org
	 */
	public void setId_org(String Id_org) {
		setAttrVal("Id_org", Id_org);
	}
	/**
	 * 受邀机构
	 * @return String
	 */
	public String getName_org() {
		return ((String) getAttrVal("Name_org"));
	}
	/**
	 * 受邀机构
	 * @param Name_org
	 */
	public void setName_org(String Name_org) {
		setAttrVal("Name_org", Name_org);
	}
	/**
	 * 受邀科室id
	 * @return String
	 */
	public String getId_dep_emp() {
		return ((String) getAttrVal("Id_dep_emp"));
	}
	/**
	 * 受邀科室id
	 * @param Id_dep_emp
	 */
	public void setId_dep_emp(String Id_dep_emp) {
		setAttrVal("Id_dep_emp", Id_dep_emp);
	}
	/**
	 * 受邀科室
	 * @return String
	 */
	public String getName_dep_emp() {
		return ((String) getAttrVal("Name_dep_emp"));
	}
	/**
	 * 受邀科室
	 * @param Name_dep_emp
	 */
	public void setName_dep_emp(String Name_dep_emp) {
		setAttrVal("Name_dep_emp", Name_dep_emp);
	}
	/**
	 * 受邀职称编码
	 * @return String
	 */
	public String getSd_emp_title() {
		return ((String) getAttrVal("Sd_emp_title"));
	}
	/**
	 * 受邀职称编码
	 * @param Sd_emp_title
	 */
	public void setSd_emp_title(String Sd_emp_title) {
		setAttrVal("Sd_emp_title", Sd_emp_title);
	}
	/**
	 * 受邀职称id
	 * @return String
	 */
	public String getId_emp_title() {
		return ((String) getAttrVal("Id_emp_title"));
	}
	/**
	 * 受邀职称id
	 * @param Id_emp_title
	 */
	public void setId_emp_title(String Id_emp_title) {
		setAttrVal("Id_emp_title", Id_emp_title);
	}
	/**
	 * 受邀职称
	 * @return String
	 */
	public String getName_emp_title() {
		return ((String) getAttrVal("Name_emp_title"));
	}
	/**
	 * 受邀职称
	 * @param Name_emp_title
	 */
	public void setName_emp_title(String Name_emp_title) {
		setAttrVal("Name_emp_title", Name_emp_title);
	}
	/**
	 * 受邀人id
	 * @return String
	 */
	public String getId_emp_doctor() {
		return ((String) getAttrVal("Id_emp_doctor"));
	}
	/**
	 * 受邀人id
	 * @param Id_emp_doctor
	 */
	public void setId_emp_doctor(String Id_emp_doctor) {
		setAttrVal("Id_emp_doctor", Id_emp_doctor);
	}
	/**
	 * 受邀人
	 * @return String
	 */
	public String getName_emp_doctor() {
		return ((String) getAttrVal("Name_emp_doctor"));
	}
	/**
	 * 受邀人
	 * @param Name_emp_doctor
	 */
	public void setName_emp_doctor(String Name_emp_doctor) {
		setAttrVal("Name_emp_doctor", Name_emp_doctor);
	}
	/**
	 * 应答时间
	 * @return FDateTime
	 */
	public FDateTime getDt_response() {
		return ((FDateTime) getAttrVal("Dt_response"));
	}
	/**
	 * 应答时间
	 * @param Dt_response
	 */
	public void setDt_response(FDateTime Dt_response) {
		setAttrVal("Dt_response", Dt_response);
	}
	/**
	 * 应答标志
	 * @return FBoolean
	 */
	public FBoolean getFg_response() {
		return ((FBoolean) getAttrVal("Fg_response"));
	}
	/**
	 * 应答标志
	 * @param Fg_response
	 */
	public void setFg_response(FBoolean Fg_response) {
		setAttrVal("Fg_response", Fg_response);
	}
	/**
	 * 应答人id
	 * @return String
	 */
	public String getId_emp_response() {
		return ((String) getAttrVal("Id_emp_response"));
	}
	/**
	 * 应答人id
	 * @param Id_emp_response
	 */
	public void setId_emp_response(String Id_emp_response) {
		setAttrVal("Id_emp_response", Id_emp_response);
	}
	/**
	 * 应答人
	 * @return String
	 */
	public String getName_emp_response() {
		return ((String) getAttrVal("Name_emp_response"));
	}
	/**
	 * 应答人
	 * @param Name_emp_response
	 */
	public void setName_emp_response(String Name_emp_response) {
		setAttrVal("Name_emp_response", Name_emp_response);
	}
	/**
	 * 会诊项目id
	 * @return String
	 */
	public String getId_srv() {
		return ((String) getAttrVal("Id_srv"));
	}
	/**
	 * 会诊项目id
	 * @param Id_srv
	 */
	public void setId_srv(String Id_srv) {
		setAttrVal("Id_srv", Id_srv);
	}
	/**
	 * 会诊项目
	 * @return String
	 */
	public String getName_srv() {
		return ((String) getAttrVal("Name_srv"));
	}
	/**
	 * 会诊项目
	 * @param Name_srv
	 */
	public void setName_srv(String Name_srv) {
		setAttrVal("Name_srv", Name_srv);
	}
	/**
	 * 会诊类型编码
	 * @return String
	 */
	public String getSd_constp() {
		return ((String) getAttrVal("Sd_constp"));
	}
	/**
	 * 会诊类型编码
	 * @param Sd_constp
	 */
	public void setSd_constp(String Sd_constp) {
		setAttrVal("Sd_constp", Sd_constp);
	}
	/**
	 * 会诊类型id
	 * @return String
	 */
	public String getId_constp() {
		return ((String) getAttrVal("Id_constp"));
	}
	/**
	 * 会诊类型id
	 * @param Id_constp
	 */
	public void setId_constp(String Id_constp) {
		setAttrVal("Id_constp", Id_constp);
	}
	/**
	 * 会诊类型
	 * @return String
	 */
	public String getName_constp() {
		return ((String) getAttrVal("Name_constp"));
	}
	/**
	 * 会诊类型
	 * @param Name_constp
	 */
	public void setName_constp(String Name_constp) {
		setAttrVal("Name_constp", Name_constp);
	}
	/**
	 * 版本号
	 * @return FDateTime
	 */
	public FDateTime getSv() {
		return ((FDateTime) getAttrVal("Sv"));
	}
	/**
	 * 版本号
	 * @param Sv
	 */
	public void setSv(FDateTime Sv) {
		setAttrVal("Sv", Sv);
	}
	/**
	 * 医疗单来源
	 * @return Integer
	 */
	public Integer getEu_sourcemd() {
		return ((Integer) getAttrVal("Eu_sourcemd"));
	}
	/**
	 * 医疗单来源
	 * @param Eu_sourcemd
	 */
	public void setEu_sourcemd(Integer Eu_sourcemd) {
		setAttrVal("Eu_sourcemd", Eu_sourcemd);
	}
	/**
	 * 费用标识
	 * @return FBoolean
	 */
	public FBoolean getFg_bl() {
		return ((FBoolean) getAttrVal("Fg_bl"));
	}
	/**
	 * 费用标识
	 * @param Fg_bl
	 */
	public void setFg_bl(FBoolean Fg_bl) {
		setAttrVal("Fg_bl", Fg_bl);
	}
	/**
	 * 临床标识
	 * @return FBoolean
	 */
	public FBoolean getFg_or() {
		return ((FBoolean) getAttrVal("Fg_or"));
	}
	/**
	 * 临床标识
	 * @param Fg_or
	 */
	public void setFg_or(FBoolean Fg_or) {
		setAttrVal("Fg_or", Fg_or);
	}
	/**
	 * 服务来源ID
	 * @return String
	 */
	public String getId_srv_src() {
		return ((String) getAttrVal("Id_srv_src"));
	}
	/**
	 * 服务来源ID
	 * @param Id_srv_src
	 */
	public void setId_srv_src(String Id_srv_src) {
		setAttrVal("Id_srv_src", Id_srv_src);
	}
	/**
	 * 计价方式
	 * @return String
	 */
	public String getPriby() {
		return ((String) getAttrVal("Priby"));
	}
	/**
	 * 计价方式
	 * @param Priby
	 */
	public void setPriby(String Priby) {
		setAttrVal("Priby", Priby);
	}
}